Sources/Datadog/DatadogConfiguration.swift

```diff
@@ -126,13 +117,10 @@ extension Datadog {
 
             internal var url: String {
                 switch self {
-                case .us1: return "https://logs.browser-intake-datadoghq.com/v1/input/"
+                case .us1, .us: return "https://logs.browser-intake-datadoghq.com/v1/input/"
                 case .us3: return "https://logs.browser-intake-us3-datadoghq.com/v1/input/"
-                case .eu1: return "https://mobile-http-intake.logs.datadoghq.eu/v1/input/"
-                case .us1_fed: return "https://logs.browser-intake-ddog-gov.com/v1/input/"
-                case .us: return "https://mobile-http-intake.logs.datadoghq.com/v1/input/"
```

Comment:
This new value of `.us` is different after this change, but this was inspired by this https://github.com/DataDog/dd-sdk-android/pull/632#discussion_r649731331 discussion in https://github.com/DataDog/dd-sdk-android/pull/632, where the new value is meant to be recommended one. @xgouchet would be good to have your look on this.